src/iframe-helper.js

```diff
@@ -36,6 +36,9 @@ export function listen(iframe, typeOfMessage, callback, opt_is3P) {
   const sentinel = getSentinel_(opt_is3P);
   let unlisten;
   let listener = function(event) {
+    const data = typeof event.data == 'string'
```

Comment:
Oh, and you will need to wrap the JSON parse in a try block. Unfortunately we get a lot of crap messages and the `{` check might not be enough.